akomlik/02_GIT.md

```diff
@@ -0,0 +1,54 @@
+  
+  ## Home Task 02_GIT.md
```

Comment:
Could you please remove whitespace before header

Reply:
I removed witespace

Reply:
I removed witespace